### PR TITLE
feat: use docker for spinning up the demo db

### PR DESCRIPTION
**Pull request by dkirchhof**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -261,7 +261,9 @@ To actively develop or debug this plugin you can either work directly within the
    This repo includes a fully working, self-seeding instance of Payload that installs the plugin directly from the source code. This is the easiest way to get started. To spin up this demo, follow these steps:
 
    1. First clone the repo
-   1. Then, `cd YOUR_PLUGIN_REPO && yarn && cd demo && cp env.example .env && yarn && yarn dev`
+   1. Then, `cd YOUR_PLUGIN_REPO && yarn && cd demo && cp .env.example .env && yarn`
```
Comment (dkirchhof) [inline README.md:264]: Btw.: cp `.env.example` instead of `cp env.example`

File: README.md [hunk 2]
```diff
@@ -261,7 +261,9 @@ To actively develop or debug this plugin you can either work directly within the
...
+   1. Start the database - You can use docker `cd YOUR_PLUGIN_REPO/demo && docker compose up` or a locally installed mongodb instance.
```
Comment (jacobsfletch) [inline README.md:265]: Need to capitalize the plural nouns here, but also bullet 2 already instructs the user to `cd` into the demo directory so no need for that.
```suggestion
   1. Start the database - You can either use Docker `docker compose up` or a locally installed MongoDB instance
```

File: README.md [hunk 3]
```diff
@@ -261,7 +261,9 @@ To actively develop or debug this plugin you can either work directly within the
...
+   1. Start payload `cd YOUR_PLUGIN_REPO/demo && yarn dev`
```
Comment (jacobsfletch) [inline README.md:266]: Need to capitalize `Payload` here but also the user is already in the demo directory, so we can simply run `yarn dev` at this step.

```suggestion
   1. Start Payload `yarn dev`
```

**Comment by jacobsfletch** [comment]
@dkirchhof, thank you for doing this! I left a couple notes of feedback. Once those are addressed we can get this merged in.

**Comment by dkirchhof** [comment]
Hey, sorry for my late response and thank you for your feedback. Merged it.

**Comment by jacobsfletch** [comment]
This plugin is now being maintained in the [Packages Directory](https://github.com/payloadcms/payload/tree/main/packages/plugin-form-builder) of the [Payload Monorepo](https://github.com/payloadcms/payload). This repo will soon be archived and all open PRs including this one will be closed. Would you be willing to [open a new PR within the Payload monorepo](https://github.com/payloadcms/payload/compare)?

**Pull request closed without merging**